include/onnxruntime/core/framework/float16.h

```diff
@@ -109,6 +109,19 @@ struct BFloat16 {
 #endif
 };
 
+// User defined suffixes to make it easier to declare
+// initializers with MLFloat16 and BFloat16 from unsigned short
+// E.g 10_f16 or 10_b16
+#if !defined(__CUDACC__) && !defined(__HIPCC__)
+inline constexpr MLFloat16 operator"" _f16(unsigned long long int v) {
```

Comment:
>unsigned long long int v [](http://example.com/codeflow?start=43&length=24)

does it just truncate values too large for uint16_t?

Reply:
I guess it does. Will change to gsl::narrow